lib/src/data/model/genre_model.dart

```diff
@@ -0,0 +1,33 @@
+class GenreModel {
+  late int id;
+  late String name;
+
+  GenreModel({
+    required this.id,
+    required this.name,
+  });
+
+  factory GenreModel.fromJson(
+    Map<String, dynamic> genreData,
+  ) {
+    return GenreModel(
+      id: genreData['id'],
+      name: genreData['name'],
+    );
+  }
+
+  static List<GenreModel> listOfGenres(
+    List<dynamic> data,
+  ) {
+    return data
+        .map(
+          (
+            genre,
+          ) =>
+              GenreModel.fromJson(
+            genre,
+          ),
+        )
+        .toList();
+  }
```

Comment:
Why GenreModel has this method?

Reply:
Because in the GenrePageModel, I need the results to be a List<GenreModel>, and the data from the json is a List<dynamic>